docs/installation/SUSE.md

```diff
@@ -8,77 +8,89 @@ parent = "smn_linux"
-to its current limitations Docker is able to run only **64 bit** architecture.
+This page provides instructions for installing and configuring the lastest
+Docker Engine software on openSUSE and SUSE systems.
 
-Docker is not part of the official repositories of openSUSE 12.3 and
-openSUSE 13.1. Hence  it is necessary to add the [Virtualization
-repository](https://build.opensuse.org/project/show/Virtualization) from
-[OBS](https://build.opensuse.org/) to install the `docker` package.
+>**Note:** You can also find bleeding edge Docker versions inside of the repositories maintained by the [Virtualization:containers project](https://build.opensuse.org/project/show/Virtualization:containers) on the [Open Build Service](https://build.opensuse.org/). This project delivers also other packages that are related with the Docker ecosystem (for example, Docker Compose).
 
-Execute one of the following commands to add the Virtualization repository:
+## Prerequisites
 
-    # openSUSE 12.3
-    $ sudo zypper ar -f http://download.opensuse.org/repositories/Virtualization/openSUSE_12.3/ Virtualization
+You must be running a 64 bit architecture.
 
-    # openSUSE 13.1
-    $ sudo zypper ar -f http://download.opensuse.org/repositories/Virtualization/openSUSE_13.1/ Virtualization
+## openSUSE
 
-No extra repository is required for openSUSE 13.2 and later.
+Docker is part of the official openSUSE repositories starting from 13.2. No
+additional repository is required on your system.
 
-# SUSE Linux Enterprise
+## SUSE Linux Enterprise
 
-Docker is available in **SUSE Linux Enterprise 12 and later**. Please note that
-due to its current limitations Docker is able to run only on **64 bit**
-architecture.
+Docker is officially supported on SUSE Linux Enterprise 12 and later. You can find the latest supported Docker packages inside the `Container` module. To enable this module, do the following:
 
-## Installation
+1. Start YaST, and select *Software > Software Repositories*.
+2. Click *Add* to open the add-on dialog.
+3. Select *Extensions and Module from Registration Server* and click *Next*.
+4. From the list of available extensions and modules, select *Container Module* and click *Next*.
+   The containers module and its repositories are added to your system.
+5. If you use Subscription Management Tool, update the list of repositories at the SMT server.
 
-Install the Docker package.
+Otherwise execute the following command:
+
+    $ sudo SUSEConnect -p sle-module-containers/12/x86_64 -r ''
+
+>**Note:** currently the `-r ''` flag is required to avoid a known limitation of `SUSEConnect`.
+
+The [Virtualization:containers project](https://build.opensuse.org/project/show/Virtualization:containers)
+on the [Open Build Service](https://build.opensuse.org/) contains also bleeding
+edge Docker packages for SUSE Linux Enterprise. However these packages are
+**not supported** by SUSE.
+
+### Install Docker
+
+1. Install the Docker package:
 
     $ sudo zypper in docker
```

Comment:
Oh! Because this is now part of a numbered list (`1. Install…`), you need to increase the indentation-level, otherwise Markdown doesn't render this as a "code" block (see the screenshot below); Indent with 8 spaces, to make it work again.

![screen shot 2015-09-03 at 17 04 32](https://cloud.githubusercontent.com/assets/1804568/9662127/6bd635be-525e-11e5-9588-bfc07f9ff948.png)